CHANGES.rst

```diff
@@ -2,6 +2,22 @@
 This describes the most significant changes. For more detail, see the commit
 log in the source code repository.
 
+What's new in v1.1
+==================
+
+- Supported PyPI binary distribution for mac and linux
```

Comment:
```suggestion
- Supported PyPI binary distributions for MacOS and Linux
```

- maybe also add the arches we build for here
- maybe add a link to the pypi package